docs/src/test/scala/docs/http/scaladsl/server/FullSpecs2TestKitExampleSpec.scala

```diff
@@ -53,8 +53,8 @@ class FullSpecs2TestKitExampleSpec extends Specification with Specs2RouteTest {
     "return a MethodNotAllowed error for PUT requests to the root path" in {
       // tests:
       Put() ~> Route.seal(smallRoute) ~> check {
-        status === StatusCodes.MethodNotAllowed
-        responseAs[String] === "HTTP method not allowed, supported methods: GET"
+        status shouldEqual StatusCodes.MethodNotAllowed
```

Comment:
yeah, you are right in this case... there was a problem in `class FullTestKitExampleSpec extends WordSpec with Matchers with ScalatestRouteTest`(in 10.0.4) where, it took `===` from `rg.scalactic.TripleEqualsSupport.Equalizer` which just returns `boolean` and doesn't fail the test. 
I was not attentive enough so in case of `FullSpecs2TestKitExampleSpec` it is taken from `org.specs2.matcher.TypedEqual.TypedEqualExpectation` which is right and works fine. 

Nevetheless, I would suggest to use `shouldEqual`, that is not implemented so often as `===` in different libs and won't confuse people who can accidentally use wrong expression in tests. For example, if they would just copy this check in their existing code that implements `WordSpec with Matchers with ScalatestRouteTest` it will actually check nothing.